doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14086,6 +14107,375 @@ AirConditioner:VariableRefrigerantFlow,
+
+#### Field: Outdoor Unit Reference Subcooling 
+
+This numeric field defines the reference subcooling degrees of the outdoor unit. If this field is blank, the default value of 3.0ºC is used. 
+
+#### Field: Refrigerant Temperature Control Algorithm for Indoor Unit
+
+This alpha field specifies the algorithm for the refrigerant temperature control. Two choices are available: ConstantTemp or VariableTemp. The indoor unit evaporating temperature at cooling mode or condensing temperature at heating are fixed in the ConstantTemp algorithm, while in VariableTemp algorithm they can be varied.
+
+#### Field: Reference Evaporating Temperature for Indoor Unit 
+
+This numeric field defines the reference evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 6.0ºC is used.
+
+#### Field: Reference Condensing Temperature for Indoor Unit 
+
+This numeric field defines the reference condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 44.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 4.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 13.0ºC is used.
+
+#### Field: Variable Condensing Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 42.0ºC is used.
+
+#### Field: Variable Condensing Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 46.0ºC is used.
+
+#### Field: Outdoor Unit Fan Object Type
+
+This alpha field defines the type of fan in the VRF outdoor unit. The only valid choices are *Fan:OnOff* and *Fan:ConstantVolume*. The input requirements for these fan objects are described elsewhere in this document.
+
```

Comment:
The curves aren't normalized?  <major-big-sigh>  That's not like 
anything else in EnergyPlus (except the refrigeration).  So, if someone 
wants to use this model, they have to simulate once to find out the 
design load, then scale all the curves to provide enough capacity?  How 
is that useable?
@hongtz68
With any kind of equipment the curves vary for different size units, but 
we've always strived to make the equipment scalable.